app/__init__.py

```diff
@@ -25,3 +27,35 @@ def resume():
 @app.route('/contact')
 def contact(): 
     return render_template('contact.html', title="Contact", url=os.getenv("URL"))
+
+@app.route('/send-email', methods=['GET','POST'])
+def send_email():
+    # HTTP POST Request args
```

Comment:
I think we should add a try/except so the site doesn't crash if the email sending messes up. We could save the success response as a variable and change it to an error message in the except block.